### ISSUE TITLE
In Container view, render uncontained processes as distinct pseudonodes

### ISSUE THREAD

**Issue by tomwilkie** [weaveworks/scope#127]

**Comment by peterbourgon** [comment]
"Not contained" is much less elegant :stuck_out_tongue:

**Comment by tomwilkie** [comment]
Given the discussion re: pseudo nodes ("everything in a topology should be
the same kind") I'm inclined to just remove uncontainer.  Especially when
we have a real containers topology with all containers in it.

**Comment by peterbourgon** [comment]
I don't understand the motivation. The proposal is to remove information from the UI, which undermines a lot of our user stories. Concretely: you **want** to see if a container is talking to more than just other containers. That single bit of information tells you a lot — it can identify misconfiguration, false assumptions you've made about your infrastructure, even identify backdoors or broken third-party software.

**Comment by tomwilkie** [comment]
A large part of good UX is consistency; making everything in a single topology the same type make explaining what it is your showing to users easier (ideally to the extend where you don't need to explain).  Exceptions (they are all containers, except that node represents incoming stuff from the internet, and that node represents stuff not in a container) make the UI less intuitive.

I agree the use cases you've highlighted are important ones; perhaps there is an alternative?  Perhaps uncontained workloads should be pseudo nodes (not all grouped into one, perhaps per-node), and perhaps we should revisit your suggestion of rendering pseudo nodes differently (smaller, greyed out...).

**Comment by peterbourgon** [comment]
I'd be happy to explode the single Uncontained node into multiple per-host nodes. And I think we should definitely render all pseudo-nodes differently. Filed #131.

**Comment by peterbourgon** [comment]
Are we OK closing this in favor of #131?

**Comment by tomwilkie** [comment]
I think there are a few things that need doing before we can close that
aren't covered by #131:
- break out uncontained into individual, per container/host pseudo nodes
- bring back 'normal' pseudo nodes for this view.

I'm kinda waiting in your big PR before I have a crack at this.
